build.gradle

```diff
@@ -1,29 +1,36 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-
 buildscript {
-    ext.kotlin_version = '1.2.20'
+    ext.kotlin_version = "1.2.20"
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath "com.android.tools.build:gradle:3.0.1"
+        //noinspection DifferentKotlinGradleVersion
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
+        classpath "org.jacoco:org.jacoco.core:0.8.0"
     }
 }
 
+plugins {
+    id "io.gitlab.arturbosch.detekt" version "1.0.0.RC6-3"
+}
+
+apply plugin: "jacoco"
+
+jacoco {
+    toolVersion = "0.8.0"
+}
+
 allprojects {
     repositories {
         google()
         jcenter()
     }
 }
```

Comment:
Maybe if the block of allProjects was before the jacoco plugin

Reply:
Done